### PR TITLE
Plug-in review guidelines

### PR DESCRIPTION
**Pull request by janjaap**
Documentation added that is specific to reviewing third-party Wordpress plug-ins.

### CODE REVIEW

**Comment by janjaap** [comment]
@Grezvany13 Could you also have a look if you think the documentation is complete?

File: reviews/plugins/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+<br />A plug-in's code should be reviewable or, when the plug-in has to be purchased, at least be available beforehand.
+
+## Review
+Once a plug-in has been selected and this plug-in is not listed in the [collection of approved and already deployed plug-ins](/#todo), the plug-in has to be reviewed by the team's lead or the CTO.
+
+### Approval
+If a plug-in is approved, it can be deployed and [the list of approved and already deployed plug-ins](/#todo) has to be updated by the approver.
+
+### Disapproval
+If it so happens that a proposed plug-in is disapproved, there are two possibilities:
```
Comment (Grezvany13) [inline reviews/plugins/README.md]: Perhaps a third option (in addition to the second one);

- If the plugin is no longer maintained, however the feature is required and code is acceptable, it is possible to request ownership on the plugin as a whole. (more details on how and who will be supplied)

File: reviews/plugins/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+- Ratings
+<br />The plug-in has to have a sufficient amount of ratings so that the average rating can be considered reliable. The amount of lowest possible ratings (1 and 2) should be minimal.
+
+- Reviews
+<br />Users that downloaded the plug-in should have posted positive reviews.
+    
+- Downloads
+<br />The plug-in should have a regular and consistent download frequency.
+
```
Comment (Grezvany13) [inline reviews/plugins/README.md]: please add:

- Compatibility
<br />The plugin must be compatible with the latest (stable) version of WordPress, and not interfere with any other plugin(s) that are used. If it does has compatibility issues with other plugins it must be clear which plugins are and why, or being able to inform you.

**Comment by Grezvany13** [comment]
A general note: please write WordPress (with capital W and P), it's the official way of writing.

**Comment by janjaap** [comment]
@Grezvany13 Could you merge this in if you agree on the contents?

**Pull request merged**